pkg/storage/elasticsearch.go

```diff
@@ -206,11 +207,14 @@ func (es ElasticSearch) GetEvents(filter *EventFilter, tenantID string) ([]*cadf
 		From(offset).Size(limit)
 
 	searchResult, err := esSearch.Do(context.Background()) // execute
-	// errcheck already within an errchecek, this is for additional detail.
 	if err != nil {
-		e, _ := err.(*elastic.Error)             //nolint:errcheck,errorlint
-		errdetails, _ := json.Marshal(e.Details) //nolint:errcheck
-		log.Printf("Elastic failed with status %d and error %s.", e.Status, errdetails)
+		var elasticErr *elastic.Error
+		if errors.As(err, &elasticErr) {
```

Comment:
FYI, we have https://pkg.go.dev/github.com/sapcc/go-bits/errext#As to make this read a bit nicer.